base/applications/sc/print.c

```diff
@@ -1,10 +1,10 @@
 /*
  * PROJECT:     ReactOS Services
- * LICENSE:     GPL - See COPYING in the top level directory
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
  * FILE:        base/applications/sc/print.c
```

Comment:
Technically speaking the new code header style does not require anymore the "FILE:" line (see ["File Structure"](https://www.reactos.org/wiki/Coding_Style#File_Structure)), so you can either keep it or remove it (I don't really mind).